Turn the ProofGeneral-specific infoH tactic into a command. (#15116)
coq/coq: doc/changelog/07-vernac-commands-and-options/15116-make-infoH-a-command.rst

- **Changed:**
  The ProofGeneral specific tactical infoH was turned into

>   a vernacular command

Reviewer: IMHO this changelog is super obscure for users, maybe describing what the impact it should be more appropiate?

Author: `infoH` is only used internally by PG, I am not sure it's worth documenting further the thing.

Reviewer: I just mean that the current message is ininteligible by users, removing it is better.

Reviewer: IMHO

Reviewer 2: ah, unintelligible

>   The ProofGeneral specific tactical infoH was turned into
>   a vernacular command

Reviewer 2: ```suggestion
  The ProofGeneral specific tactic infoH was replaced
  by the :cmd:`InfoH` command
```

The documentation avoids the term "tactical" and "vernacular command".
